Translation corrections (#290)
07th-mod/higurashi-console-arcs: Update/toki_007.txt

			NULL, " It seems to have built up due to overwork.", Line_WaitForInput);
	ModPlayVoiceLS(4, 10, "ps3/s11/10/420900012", 256, TRUE);
	OutputLine(NULL, "ここしばらく、あまり十分に寝ておられないんじゃないですか？」",

> 			NULL, " You haven't gotten a lot of sleep recently, have you?\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " You haven't been getting much sleep recently, have you?\"", Line_Normal);
```

			NULL, "\"Well, I should get going soon.", Line_WaitForInput);
	ModPlayVoiceLS(4, 28, "ps3/s11/28/420100959", 256, TRUE);
	OutputLine(NULL, "ごめんね、好き勝手に喋って時間もらっちゃって」",

> 			NULL, " Sorry for taking up your time by chatting whatever I thought.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " Sorry for taking up your time chatting about whatever I felt like.\"", Line_Normal);
```
